expert-forecast/expert-forecast-app/app.R

```diff
@@ -202,9 +220,9 @@ server <- function(input, output, session) {
         
         if(input$f_id %in% check_ids$id){
             
-            submit_df <- cbind(as.character(unique(raw_data$target_end_date)), rep(input$location, 4), rep("ee0.5", 4), c(input$pt_wk1, input$pt_wk2, input$pt_wk3, input$pt_wk4)) %>%
-                rbind(cbind(as.character(unique(raw_data$target_end_date)), rep(input$location, 4), rep("ee0.05", 4), c(input$qt_wk1[1], input$qt_wk2[1], input$qt_wk3[1], input$qt_wk4[1]))) %>%
-                rbind(cbind(as.character(unique(raw_data$target_end_date)), rep(input$location, 4), rep("ee0.95", 4), c(input$qt_wk1[2], input$qt_wk2[2], input$qt_wk3[2], input$qt_wk4[2]))) %>%
+            submit_df <- cbind(as.character(unique(rt_data$target_end_date)), rep(input$location, 4), rep("ee0.5", 4), c(input$pt_wk1, input$pt_wk2, input$pt_wk3, input$pt_wk4)) %>%
+                rbind(cbind(as.character(unique(rt_data$target_end_date)), rep(input$location, 4), rep("ee0.05", 4), c(input$qt_wk1[1], input$qt_wk2[1], input$qt_wk3[1], input$qt_wk4[1]))) %>%
```

Comment:
this looks like it works but is a bit of a mission! Not clear on how to refactor though so not the most helpful of comments!